tests/testsuite/support/mod.rs

```diff
@@ -990,6 +1003,15 @@ impl Execs {
     }
 }
 
+impl Drop for Execs {
+    fn drop(&mut self) {
+        if !self.ran {
+            // TODO: Re-enable when everything goes through Execs#run
```

Comment:
Overall I like the direction. mabe add a `#[must_use]` to `Execs`?

Reply:
I'm copying Alex's implementation in rustfix: https://github.com/rust-lang-nursery/rustfix/blob/116c7e9f7b666d92c049ab86c1d83e8ced1cead4/cargo-fix/tests/all/main.rs#L323-L329

How do the two techniques compare?

Reply:
Interesting: `ProjectBuilder` already has the `must_use` attribute applied to it.

Reply:
Defence in depth, use both. One checks at runtime the other at compile time.